2022/2023-01-17_europe_africa_cc.md

```diff
@@ -0,0 +1,35 @@
+# scikit-image Community Call
+
+- **Time:** Tuesday, January 17, 2023, 17:00 – 18:00 UTC
+- **[Join video call via Zoom](https://us06web.zoom.us/j/88060567580?pwd=THRpaWFnSFNwK0Fycy9FVk5RYnV5UT09)**
+- **Notes:** https://hackmd.io/@scientific-python/HJWUBDVNi
+- **[scikit-image Community Calendar](https://scientific-python.org/calendars/skimage.ics)**
+- **[Archive — Meeting Notes](https://github.com/scikit-image/meeting-notes)**
+- **[Code of Conduct](https://scikit-image.org/docs/stable/conduct/code_of_conduct.html)**
+
+**Present:** Marianne, Lars, Stéfan
+
+**Moderator:** -
+
+## Agenda
+
+- [HackMD doc by Scientific Python](https://hackmd.io/@scientific-python/HJWUBDVNi) is readonly?
+	- Message "Sorry, this team is disabled. You can't edit this note."
+	- HackMD agreed to upgrade our account again; Stéfan emailed them to find out when this will happen
+- Respond to [New (experimental) type stubs for skimage](https://discuss.scientific-python.org/t/621)
+	- CZI5's goal is to type scikit-image
+	- might be a useful starting point / reference to keep in mind
+- Release 0.20
+	- Blocked by [Build wheels for arm64 on macos
```

Comment:
We should probably set indentation=2 in the editor!

Reply:
Agreed, I'll update it.